Description: fix build directory for test runner
 Hardcode Python 3 build directory in the test runner to the one that Debian
 package uses.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2012-02-12

--- a/tests/run.py
+++ b/tests/run.py
@@ -14,11 +14,11 @@
 from os import path, chdir, listdir
 
 if sys.version_info >= (3, 0):
-    print('Copying and converting sources to build/lib/tests...')
+    print('Copying and converting sources to build/py3/tests...')
     from distutils.util import copydir_run_2to3
     testroot = path.dirname(__file__) or '.'
     newroot = path.join(testroot, path.pardir, 'build')
-    newroot = path.join(newroot, listdir(newroot)[0], 'tests')
+    newroot = path.join(newroot, 'py3', 'tests')
     copydir_run_2to3(testroot, newroot)
     # switch to the converted dir so nose tests the right tests
     chdir(newroot)
